### PR TITLE
Add note about conflicting `runserver` commands

### PR DESCRIPTION
**Pull request by lwm**
My RST may be rusty.

Refs:
  * https://github.com/evansd/whitenoise/issues/77
  * https://github.com/django/channels/issues/684

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -20,6 +20,14 @@ Once that's done, you should add ``channels`` to your
 That's it! Once enabled, ``channels`` will integrate itself into Django and
 take control of the ``runserver`` command. See :doc:`getting-started` for more.
 
+.. note::
+  Please be wary of any other app that require an overloaded or
```
Comment (andrewgodwin) [inline docs/installation.rst]: Could you change "app" to "third-party apps"?

File: docs/installation.rst [hunk 2]
```diff
@@ -20,6 +20,14 @@ Once that's done, you should add ``channels`` to your
...
+  replacement ``runserver`` command. Channels provides a separate
+  ``runserver`` command and may conflict with it. An example
+  of such a conflict is with `whitenoise.runserver_nostatic <https://github.com/evansd/whitenoise/issues/77>`
+  from the `whitenoise <https://github.com/evansd/whitenoise>` app.
```
Comment (andrewgodwin) [inline docs/installation.rst]: Could you add a paragraph about how to resolve the issue (either moving `channels` to the top of `INSTALLED_APPS`, or removing the offending app?)

**Comment by andrewgodwin** [comment]
Added a couple of comments, and you're missing the `_` after the links in RST (you can view the RST file directly on your branch in github to check things like that)

**Comment by lwm** [comment]
Fixed it up based on your comments, looks good to go?

**Comment by andrewgodwin** [comment]
Lovely, thanks!

**Pull request merged**